src/browser/modules/D3Visualization/components/NodeInspectorPanel.tsx

```diff
@@ -70,44 +68,40 @@ export class NodeInspectorPanel extends Component<NodeInspectorPanelProps> {
           )}
         </StyledNodeInspectorTopMenuChevron>
 
-        {expanded && (
-          <StyledNodeInspectorContainer
+        <NodeInspectorDrawer width={width} openDrawer={expanded}>
```

Comment:
Got a bit confused here because we've used "openDrawer" to be a drawer name in the past, anyway I think `isOpen` would be a better name, but openDrawer also works fine